views/celebrities/celebrities.hbs

```diff
@@ -0,0 +1,11 @@
+<h2>List of our celebrities</h2><br>
+
+{{#each celebritiesArr}}
+    <section>
+        <h3>{{this.name}}</h3>
+        <p>
+            Occupation: {{this.occupation}} <br><br>
+            Catch-Phrase: {{this.catchPhrase}} <br>
+        </p>
+    </section>
+{{/each}} 
```

Comment:
Here it is nt neccesary to use keyword `this`. We need it if we the variable (the data structure) that we send from js to view has no name. 

In this case you are iterating through celebritties array, and in each iteration you are printing details one signle celeb object and you can access properties directly